src/components/Tag/Tag.tsx

```diff
@@ -62,9 +62,8 @@ function Tag({ color, children, className, icon, variant = "flat" }: Props) {
         color && stylesByColor[color],
         variant === "outline" && styles.outline,
       )}
-      color={color}
       size="sm"
-      spacing="none"
+      variant={color === "yellow" ? undefined : color}
```

Comment:
🤔  can we just get rid of the yellow variant? this feels like sneaky / unexpected behavior

Reply:
Good q, I'm not exactly sure. In here and `main`, the color of the `Tag` itself is passed to `Text` to style the `Text` color. However, in `next`, `Text` is styled via class styling passed from within `Tag`. (not sure if this word salad is making any sense).

Also in `next`, `yellow` does not exist but `yield` and `warning` which are two yellow `Tag`s, but design work is still ongoing this and even deprecating green/go, yellow/yield/warning, red/error Tag variants.

As a result, with regards to `Text` here in `Tag` in `main`,

I'll remove passing `variant` to `Text` and just inherit the color that's already being applied in `Tag`, like in `main` to preserve this `yellow` text for now and to be more similar to `next`. The `Tag` [ticket](https://app.shortcut.com/czi-edu/story/193791/update-tag-api-in-traject) itself can be used to fully migrate `Tag` later once decisions have been completed

Reply:
ahh right right, because we're removing the `yellow` variant from Text. Thanks for explaining, that solution sgtm 👍